tests/tests.cpp

```diff
@@ -58,7 +58,7 @@ ms::NDImage<T, 2> cimg2ndimage(CImg<T>& img)
 void lakes()
 {
     // Load image
-    CImg<double> img = rgb2gray(CImg<double>("../testimages/lakes3.jpg")) / 255.0;
+    CImg<double> img = rgb2gray(CImg<double>("../images/lakes3.jpg")) / 255.0;
```

Comment:
This is OK and I'll keep it this way once a PR is accepted, but in general I would avoid renaming and moving files unless there is a good reason.